code/src/App.js

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
+import { Header } from 'components/Header';
 import data from './data.json';
-
-console.log(data);
+import Album from './components/album';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <div className="header">
+        <Header />
+      </div>
+      <div>
+        <main className="albums">
+          {data.albums.items.map((item) => {
+            return (
+              <Album
+                key={item.id}
+                name={item.name}
+                artists={item.artists}
+                image={item.images[0]}
```

Comment:
It stood out to us you picked the object from the image array at this stage and not earlier in a component. Interesting!